### ISSUE TITLE
SasView 6.0.0-beta doesn't start on Linux

### ISSUE THREAD

**Issue by wpotrzebowski** [SasView/sasview#2902]
**Describe the bug**
I've been trying to run SasView on vanilla linux machine (Rocky Linux 9.3) but I am getting following error

![Screenshot 2024-06-24 at 13 05 51](https://github.com/SasView/sasview/assets/4189922/dc87ce52-e898-4bf8-b1d1-cfe252d770be)

**Comment by krzywon** [comment]
I was seeing this same error on Windows when moving to Pyinstaller 6 and was related to how the bundling is happening. Can you try the build on #2862 to see if this has been resolved there?

**Comment by wpotrzebowski** [comment]
Indeed, it starts with #2862 but crashes badly when trying to load any data (not sure if the error message below is helpful).

```
(sasview:1322555): Gtk-WARNING **: 09:30:45.662: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Adwaita/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Adwaita/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Aborted (core dumped)
```

**Comment by wpotrzebowski** [comment]
It seems that there are some issues related to GL installation. It also crushes when I try to run Orientation Viewer
![Screenshot 2024-07-03 at 14 51 19](https://github.com/SasView/sasview/assets/4189922/3d81758e-4cac-476b-b355-f692f1bcd2fe)

**Comment by krzywon** [comment]
I'm leaving this open for now. The PR to fix this was merged, but it's still not clear if this has been resolved.

**Comment by krzywon** [comment]
@bmaranville tested the v6.0.0 Beta2 installer on Ubuntu 20.04 and sasview launched. I am closing this for now, but feel free to reopen if the launch issue remains.
